gfx/map/city_data/city_types/french_city.txt

```diff
@@ -0,0 +1,121 @@
+﻿french_city = {
```

Comment:
I think this comes from the VOTP expansion pack? If it is, I'm afraid I can't include it as it would infringe paradox's copyright which might then cause the mod to be delisted from steam.